packages/image_picker/image_picker_platform_interface/lib/src/method_channel/method_channel_image_picker.dart

```diff
@@ -143,8 +143,14 @@ class MethodChannelImagePicker extends ImagePickerPlatform {
 
   @override
   Future<LostData> retrieveLostData() async {
-    final Map<String, dynamic>? result =
-        await _channel.invokeMapMethod<String, dynamic>('retrieve');
+    Map<String, dynamic>? result;
+    List<PickedFile>? pickedFileList;
+
+    try {
+      result = await _channel.invokeMapMethod<String, dynamic>('multiRetrieve');
+    } on MissingPluginException {
+      result = await _channel.invokeMapMethod<String, dynamic>('retrieve');
+    }
```

Comment:
Don't think we need to add new features to `retrieveLostData()` anymore, because it's deprecated and users of the package should switch to `getLostData()`.